<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog</class>
 <widget class="QDialog" name="QDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>808</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HoloLens MDL2 Assets</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Search tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>window.png</normaloff>window.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>HoloLens MDL2 Assets</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>検索文字列</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>HoloLens MDL2 Assets</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>濾過文字列</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="edit_key">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>HoloLens MDL2 Assets</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filter_key">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>HoloLens MDL2 Assets</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="toolTip">
          <string>検索</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>search.png</normaloff>search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>80</height>
          </size>
         </property>
         <property name="toolTip">
          <string>終了</string>
         </property>
         <property name="statusTip">
          <string>終了</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>quit.png</normaloff>quit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>700</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>URL</string>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>QDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_search</sender>
   <signal>clicked()</signal>
   <receiver>QDialog</receiver>
   <slot>showResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_key</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QDialog</receiver>
   <slot>changeSearchKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSearchKey()</slot>
  <slot>enableBtn()</slot>
 </slots>
</ui>
